docs/docs-content/downloads/artifacts-studio.md

```diff
@@ -0,0 +1,263 @@
+   `REPLACE ME` is replaced with the path and name of the public key downloaded.
+
+   ```shell title="Example" {10}
+
+    # This script verifies the signatures of data files against a public key.
+    # It expects files named like `datafile.zst` and corresponding signatures `datafile.sig.bin`.
+    # Ensure you have OpenSSL installed to run this script and that the public key is in PEM format.
+    # Ensure the script has execute permissions: `chmod +x verify.sh`
+    # Usage: ./verify.sh
+
+    #!/bin/bash
+
+    PUBKEY="REPLACE ME" # Enter path to public key file
+
+    for sigfile in *.sig.bin; do
+      # Strip `.sig.bin` to get base filename and corresponding data file
+      base="${sigfile%.sig.bin}"
+      datafile="${base}.zst"
+
+      if [[ ! -f "$datafile" ]]; then # Check if data file exists
+        echo "$datafile: ❌ Data file not found" # Skip to next iteration
+        continue
+      fi
+
+      # Run signature verification
+      if openssl dgst -sha256 -verify "$PUBKEY" -signature "$sigfile" "$datafile" > /dev/null 2>&1; then # Verify signature
+        echo "$datafile: ✅ Signature valid" # Print success message
+      else
+        echo "$datafile: ❌ Signature invalid" # Print failure message
+      fi
+    done
+   ```
+
+   You should see the following output.
+
+   ```shell title="Output Example"
+   cni-calico-3.30.1.zst: ✅ Signature valid
+   csi-aws-ebs-1.43.0.zst: ✅ Signature valid
+   kubernetes-1.32.4.zst: ✅ Signature valid
+   spectro-k8s-dashboard-7.11.1.zst: ✅ Signature valid
+   ubuntu-aws-22.04.zst: ✅ Signature valid
+   ```
+
+## Download a Specific Pack
+
+1. Click on **Browse Packs**.
+
+   ![Image showing where to download individual packs](../../../static/assets/docs/images/downloads/downloads_artifact-studio-browse-packs.webp)
+
+2. Filter based on **Product**, **Version** (Product version), **Cloud type**, **Layer type**, CPU version and whether
+   it is FIPS compliant or not.
+
+3. Enter your search terms and click **Search** to further filter.
+
+   ![Image showing filter and search together](../../../static/assets/docs/images/downloads/downloads_artifact-studio-search-and-filter.webp)
+
+4. Select one or more packs, and click the **I'm not a robot** reCAPTCHA to download the bundle. The bundle will
+   download as individual pack files (`<filename>.zst`) and individual signature file (`<filename>.bin`). Ensure that
+   all the files are saved in the same folder. Alternatively, you click **Copy all URLs** to download the individual
+   files via the individual URLs.
+
+   ![Image showing download of multiple packs](../../../static/assets/docs/images/downloads/downloads_artifact-studio-individual-packs-download.webp)
+
+   :::tip
+
+   Ensure that pop-ups are enabled in your browser for [Artifact Studio](https://artifact-studio.spectrocloud.com/) to
+   allow for multiple artifact downloads.
+
+   :::
+
+### Specific Pack Download Verification
```

Comment:
This looks almost completely the same as the bundle verification process. Maybe use a partial to reduce duplication?